### DISCUSSION TITLE
KubernetesExecutor - how can i switch context ?

### DISCUSSION THREAD

**Discussion by itay-bardugo** [apache/airflow#25796] [Q&A]
hey! 
i use KubernetesExecutor,
and if i am running KubernetesPodOperator, it has the `cluster_context` parameter which allows me running  pods over multiple clusters.
but what if i am using PythonOperator? do i have the ability to set the `cluster_context` some how in order for me to run the operator over multiple clusters?

**Comment by potiuk** [comment]
No. Python Operator always uses the default one.

**Comment by potiuk** [comment]
However you can use @task.docker decorator likely to achieve what you want  https://airflow.apache.org/docs/apache-airflow/stable/tutorial_taskflow_api.html?highlight=docker+task#using-the-taskflow-api-with-docker-or-virtual-environments
